### PR TITLE
[ADD] product_attribute_value_related_operations

### PR DESCRIPTION
**Pull request by mikelarre**

### CODE REVIEW

File: product_attribute_value_related_operations/models/mrp_bom.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            return production.product_id.attribute_value_ids.mapped(
+                'operation_ids')
+        else:
+            return production.product_attribute_ids.mapped(
+                'value_id.operation_ids')
+
+    @api.model
+    def _skip_bom_line(self, line, product):
+        delete_product_lines = self._get_values_operations(
+            self._context.get('production'))
```
Comment (Daniel-CA) [inline product_attribute_value_related_operations/models/mrp_bom.py]: self.env.context.get('production')

File: product_attribute_value_related_operations/models/mrp_bom.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        if line.operation.operation in delete_product_lines:
+            return True
+        return super(MrpBom, self)._skip_bom_line(line, product)
+
+    @api.multi
+    def _prepare_wc_line(self, wc_use, level=0, factor=1):
+        res = super(MrpBom, self)._prepare_wc_line(wc_use, level=level,
+                                                   factor=factor)
+        production = self._context.get('production')
```
Comment (Daniel-CA) [inline product_attribute_value_related_operations/models/mrp_bom.py]: berdin

**Review by Daniel-CA — APPROVED** [review]
:+1:

**Comment by alfredoavanzosc** [comment]
:+1:

**Pull request merged**